modules/alloy/src/main/ts/ephox/alloy/behaviour/sliding/SlidingUtils.ts

```diff
@@ -1,10 +1,4 @@
 import { AlloyComponent } from '../../api/component/ComponentApi';
 import { SlidingConfig } from './SlidingTypes';
 
-export const getAnimationRoot = (component: AlloyComponent, slideConfig: SlidingConfig) => {
-  return slideConfig.getAnimationRoot.fold(() => {
-    return component.element();
-  }, (get) => {
-    return get(component);
-  });
-};
\ No newline at end of file
+export const getAnimationRoot = (component: AlloyComponent, slideConfig: SlidingConfig) => slideConfig.getAnimationRoot.fold(() => component.element(), (get) => get(component));
```

Comment:
Another one where it's all been shoved onto a single line.

Reply:
Fixed